doc/nrf/gs_modifying.rst

```diff
@@ -1,13 +1,15 @@
 .. _gs_modifying:
 
-Modifying a sample application
-##############################
+Modifying an application
+########################
 
 .. contents::
    :local:
    :depth: 2
 
-After programming and testing a sample application, you probably want to make some modifications to the application, for example, add your own files with additional functionality, change compilation options, or update the default configuration.
+For simplicity, this guide will refer to both :ref:`samples<samples>` and :ref:`applications<applications>` as "applications".
```

Comment:
You can put this into shortcuts.txt and use a replacement tag here and on the programming and testing pages.

Reply:
Updated with your suggestion.